plugins/tiff/src/tiffpresentation.hh

```diff
@@ -36,14 +36,15 @@ private:
   int height;
   int width;
   TiledBitmapInterface::Ptr tbi;
-  LayerSpec ls;
   int bps;
   int spp;
   std::map<std::string, std::string> properties;
   Views views;
   ColormapHelper::Ptr colormapHelper;
   TransformationData::Ptr transformationData;
 
+  PipetteLayerOperations::Ptr pipetteLayer;  //holds the most bottom layer for pipette operations
```

Comment:
Actually, it doesn't hold the most bottom layer, but the most bottom `LayerOperations`.

So perhaps you should remove the comment and rename the member variable?